### PR TITLE
Enhancement: Move reporting metrics to postgres

### PR DESCRIPTION
**Pull request by subintp**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/builders/v2/report_builder.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    scope.events.where(name: 'conversation_resolved')
+         .group_by_day(:created_at, range: range, default_value: 0)
+         .average(:value)
+  end
+
+  def range
+    parse_date_time(params[:since])..parse_date_time(params[:until])
+  end
+
+  # Taking average of average is not too accurate
```
Comment (sojan-official) [inline app/builders/v2/report_builder.rb]: 😆

File: app/models/agent_bot_inbox.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # Table name: agent_bot_inboxes
 #
 #  id           :bigint           not null, primary key
-#  status       :integer          default("active")
```
Comment (pranavrajs) [inline app/models/agent_bot_inbox.rb]: Why is this changed?

Comment (subintp) [reply app/models/agent_bot_inbox.rb]: Someone forgot to commit this. I committed myself.
